.evergreen.yml

```diff
@@ -358,7 +358,16 @@ functions:
       working_dir: spruce
       script: |
         echo "Generating .cmdrc.json"
-        REACT_APP_BUGSNAG_API_KEY=${REACT_APP_BUGSNAG_API_KEY} REACT_APP_SENTRY_DSN=${REACT_APP_SENTRY_DSN} REACT_APP_NEW_RELIC_ACCOUNT_ID=${REACT_APP_NEW_RELIC_ACCOUNT_ID} REACT_APP_NEW_RELIC_AGENT_ID=${REACT_APP_NEW_RELIC_AGENT_ID} REACT_APP_NEW_RELIC_APPLICATION_ID=${REACT_APP_NEW_RELIC_APPLICATION_ID} REACT_APP_NEW_RELIC_LICENSE_KEY=${REACT_APP_NEW_RELIC_LICENSE_KEY} REACT_APP_NEW_RELIC_TRUST_KEY=${REACT_APP_NEW_RELIC_TRUST_KEY} REACT_APP_DEPLOYS_EMAIL=${REACT_APP_DEPLOYS_EMAIL} node scripts/setup-credentials.js
+        REACT_APP_BUGSNAG_API_KEY=${REACT_APP_BUGSNAG_API_KEY} \
```

Comment:
Assuming you know more about evergreen ymls and  this won't break the script :)

Reply:
[I tested it out](https://parsley.mongodb.com/evergreen/spruce_ubuntu2204_deploy_to_prod_patch_72f3fa46cc285af5acd1c856151c680059d403e8_64ac7cde32f4170b31df8050_23_07_10_21_49_59/0/task?bookmarks=0,193&shareLine=138) and it appears to have worked.